src/test/kotlin/net/maizegenetics/phgv2/utils/SeqUtilsTest.kt

```diff
@@ -0,0 +1,174 @@
+
+        val tempDir = "${System.getProperty("user.home")}/temp/phgv2Tests/tempDir/"
+        @JvmStatic
+        @BeforeAll
+        fun setup() {
+            File(TestExtension.tempDir).mkdirs()
+
+            // create the agc compressed file from which we'll pull data
+            val fastaInputDir = "data/test/smallseq"
+            val fastaOutputDir = TestExtension.testOutputFastaDir
+            val dbPath = TestExtension.testTileDBURI
+
+            // Why isn't this handled in TestExtension - I have added the command "File(testTileDBURI).mkdirs()"
+            // to TestExtension
+            Files.createDirectories(Paths.get(dbPath))
+
+            // copy files with extension .fa from data/test/smallseq to fastaOutputDir
+            val fastaFiles = File(fastaInputDir).listFiles { file -> file.extension == "fa" }
+            fastaFiles.forEach { file -> file.copyTo(File(fastaOutputDir, file.name)) }
+
+
+            // get the full path fasta file names  from the fastaInput, write to fileList
+            val fileList = mutableListOf<String>()
+            File(fastaOutputDir).walk(FileWalkDirection.TOP_DOWN).filter{it.name.endsWith(".fa")}.forEach {
+                fileList.add(it.toString())
+            }
+
+            // The list of fasta files to load should not contain the reference.  The reference is listed separately.
+            fileList.removeIf { it.contains("Ref") }
+
+            // write the full path fasta file names  from the fastaOutputDir to a single file, one per line, in tempDir
+            // This file will be used as input to the agc-compress command
+            val fastaCreateFileNamesFile = File(dbPath, "fastaCreateFileNames.txt")
+            fastaCreateFileNamesFile.writeText(fileList.joinToString("\n"))
+
+            val refFasta = File(fastaOutputDir, "Ref.fa").toString()
+
+            val agcCompress = AgcCompress()
+            // Create the initial compressed file
+            println("Calling agcCompress for CREATE")
+            var result = agcCompress.test("--fasta-list ${fastaCreateFileNamesFile} --db-path ${dbPath} --ref-fasta ${refFasta}")
+
+        }
+
+        @JvmStatic
+        @AfterAll
+        fun teardown() {
+            File(TestExtension.tempDir).deleteRecursively()
+        }
+    }
+
+    @Test
+    fun testBuildAgcCommandFromList() {
+        // test command with single range on the list
+
+        val dbPath = TestExtension.testTileDBURI
+        var rangeList = mutableListOf<String>()
+        val range1 = "1@LineA:20-40"
+        rangeList.add(range1)
+        var command = buildAgcCommandFromList(dbPath,rangeList)
+
+        //The command should look like:
+        val expectedCommand = arrayOf("conda","run","-n","phgv2-conda","agc","getctg","${dbPath}/assemblies.agc",range1)
+
+        // Verify that command contains all elements from expected command and that the order is the same
+        kotlin.test.assertEquals(expectedCommand.size, command.size)
```

Comment:
These should be cleaned up.  I think the rest of the repo is using kotlin's testing calls and not JUnit5(Jupiter).  It looks like you have the Jupiter imports in which can be removed.

Reply:
done